src/config/rcxml.c

```diff
@@ -1163,63 +1163,94 @@ rcxml_init(void)
 	rc.workspace_config.min_nr_workspaces = 1;
 }
 
-static struct {
-	const char *binding, *action, *attribute, *value;
+static struct key_combos {
+	const char *binding, *action;
+	struct {
+		const char *name, *value;
+	} attributes[3];
 } key_combos[] = {
-	{ "A-Tab", "NextWindow", NULL, NULL },
-	{ "W-Return", "Execute", "command", "alacritty" },
-	{ "A-F3", "Execute", "command", "bemenu-run" },
-	{ "A-F4", "Close", NULL, NULL },
-	{ "W-a", "ToggleMaximize", NULL, NULL },
-	{ "A-Left", "MoveToEdge", "direction", "left" },
-	{ "A-Right", "MoveToEdge", "direction", "right" },
-	{ "A-Up", "MoveToEdge", "direction", "up" },
-	{ "A-Down", "MoveToEdge", "direction", "down" },
-	{ "W-Left", "SnapToEdge", "direction", "left" },
-	{ "W-Right", "SnapToEdge", "direction", "right" },
-	{ "W-Up", "SnapToEdge", "direction", "up" },
-	{ "W-Down", "SnapToEdge", "direction", "down" },
-	{ "A-Space", "ShowMenu", "menu", "client-menu"},
-	{ "XF86_AudioLowerVolume", "Execute", "command", "amixer sset Master 5%-" },
-	{ "XF86_AudioRaiseVolume", "Execute", "command", "amixer sset Master 5%+" },
-	{ "XF86_AudioMute", "Execute", "command", "amixer sset Master toggle" },
-	{ "XF86_MonBrightnessUp", "Execute", "command", "brightnessctl set +10%" },
-	{ "XF86_MonBrightnessDown", "Execute", "command", "brightnessctl set 10%-" },
-	{ NULL, NULL, NULL, NULL },
+	{ "A-Tab", "NextWindow", {
```

Comment:
Consider simplifying as follows:

```C
struct key_combos {
	const char *binding, *action;
	struct {
		const char *name, *value;
	} attributes[3];
} key_combos[] = { {
		.binding = "A-Tab",
		.action = "NextWindow",
	}, {
		.binding = "A-Space",
		.action = "ShowMenu",
		.attributes[0] = {
			.name = "menu",
			.value = "client-menu",
	       	},
		.attributes[1] = {
			.name = "atCursor",
			.value = "no",
	       	},
	}
};
```

Reply:
This is, I think, much more readable than other options. I've adopted it int he latest commit.